components/google_sheets/actions/add-single-row/add-single-row.mjs

```diff
@@ -56,59 +33,48 @@ export default {
           driveId: googleSheets.methods.getDriveId(c.drive),
         }),
       ],
+      description: "",
+      withLabel: true,
     },
     sheetName: {
       propDefinition: [
         googleSheets,
         "sheetName",
         (c) => ({
-          sheetId: c.sheetId,
+          sheetId: c.sheetId.value,
         }),
       ],
     },
-    dataFormat,
+    headerProp,
   },
   async additionalProps() {
     const props = {};
```

Comment:
I would recommend removing this and just `return`ing from the relevant sections (see below)